docs/integrating-with-unity/defining-actions.mdx

```diff
@@ -149,27 +150,263 @@ private void AttackPlayer(GameObject player)
   // your implementation here...
 }
 ```
+:::
+
+
+# Invoking an `RGAction` on a GameObject
+As mentioned above, using the `[RGAction]` attribute causes Regression Games to generate a new code class for handling invoking actions
+for that MonoBehaviour.
+
+One of the generated classes is an `RGActionRequest` which can be used by your bot code to quickly invoke an action on a specific game object.
 
+:::info
+Note that by default you can only invoke actions on GameObjects for bots owned by your client id, or on global action
+handlers registered on the Regression Games overlay.
+
+_Clicking UI buttons is a global action handler available by default on the overlay.  Clicking a button is included in the example below._
 :::
 
-## Assigning Actions to GameObjects
+### `[RGAction]` Code Generation Example
```

Comment:
Do we need to provide this? I think we should just allow the developer to assume this is abstracted. Maybe instead we can have this in a different page, or a link to an example of a generated class in one of our repos? I think we should find a balance of letting them know how it's done, but then not overloading them too much since they really don't need to know.

Reply:
Personally I'm against removing it.

Maybe we can improve the wording or transitions around it.

I started with just have the Bot action implementation part, but it lacked context for where `RGActionRequest_Player_MoveToPosition` came from or how it was defined.

Reply:
Fine with me - I'm not super opposed and we can always use feedback and data to support my claim if my assumption is true.